main.tf

```
@@ -209,12 +215,29 @@ resource "kubernetes_service" "service" {
       app = local.app_name
     }
     port {
+      name      = "http"
       port      = 8080
       node_port = var.service_port
     }
   }
 }
 
+resource "kubernetes_service" "prometheus_service" {
+  metadata {
+    name = "prometheus"
+  }
+
+  spec {
+    selector = {
+      app = local.app_name
+    }
+    port {
+      name      = "prometheus"
+      port      = 8181
```

```suggestion
      name = "prometheus"
      port   = 8181
```
could you run the terraform fmt command?